test/e2e/kubectl/kubectl.go

```diff
@@ -364,6 +366,104 @@ var _ = SIGDescribe("Kubectl client", func() {
 		})
 	})
 
+	ginkgo.Describe("kubectl get output", func() {
+		ns := "testing-kubectl-get-output-" + rand.String(10)
+		objMeta := metav1.ObjectMeta{Name: ns}
+
+		ginkgo.BeforeEach(func() {
+			_, err := c.CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: objMeta})
```

Comment:
Why you need to create that namespace? Why you can't just use `f.Namespace.Name` which is already creating a random namespace for each test case?